Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/klipper/readme/) for Klipper/readme.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/klipper/-/readme/horizontal-auto.svg)

dingyifei commented:
@math-a3k If you need admin permission on Weblate to do something, please let me know!

dingyifei committed:
locales: Translated using Weblate (Polish)

Currently translated at 4.4% (2 of 45 strings)

locales: Translated using Weblate (Italian)

Currently translated at 5.0% (1 of 20 strings)

locales: Translated using Weblate (Italian)

Currently translated at 1.0% (1 of 92 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 0.7% (1 of 140 strings)

locales: Translated using Weblate (Turkish)

Currently translated at 11.6% (5 of 43 strings)

locales: Translated using Weblate (Spanish)

Currently translated at 68.8% (31 of 45 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 89.1% (82 of 92 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 11.1% (5 of 45 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 88.0% (81 of 92 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 73.6% (14 of 19 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (28 of 28 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 39.3% (57 of 145 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (62 of 62 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 71.4% (20 of 28 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 27.2% (15 of 55 strings)

locales: Translated using Weblate (Chinese (Simplified))

Currently translated at 16.4% (13 of 79 strings)

Co-authored-by: Yifei Ding <dingyifeiair@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/klipper/beaglebone/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/klipper/bed_level/es/
Translate-URL: https://hosted.weblate.org/projects/klipper/bed_level/pl/
Translate-URL: https://hosted.weblate.org/projects/klipper/bed_level/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/klipper/bed_mesh/it/
Translate-URL: https://hosted.weblate.org/projects/klipper/bed_mesh/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/klipper/bootloaders/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/klipper/config_changes/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/klipper/contributing/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/klipper/delta_calibrate/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/klipper/endstop_phase/it/
Translate-URL: https://hosted.weblate.org/projects/klipper/installation/tr/
Translate-URL: https://hosted.weblate.org/projects/klipper/measuring_resonances/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/klipper/rpi_microcontroller/zh_Hans/
Translation: Klipper/Bed_Level
Translation: Klipper/Bed_Mesh
Translation: Klipper/Bootloaders
Translation: Klipper/CONTRIBUTING
Translation: Klipper/Config_Changes
Translation: Klipper/Delta_Calibrate
Translation: Klipper/Endstop_Phase
Translation: Klipper/Installation
Translation: Klipper/Measuring_Resonances
Translation: Klipper/RPi_microcontroller
Translation: Klipper/beaglebone

math-a3k committed:
locales: Translated using Weblate (Spanish)

Currently translated at 100.0% (31 of 31 strings)

locales: Translated using Weblate (Spanish)

Currently translated at 87.0% (27 of 31 strings)

locales: Translated using Weblate (Spanish)

Currently translated at 22.5% (7 of 31 strings)

locales: Translated using Weblate (Spanish)

Currently translated at 1.4% (1 of 69 strings)

locales: Translated using Weblate (Spanish)

Currently translated at 100.0% (1 of 1 strings)

Co-authored-by: Matemática A3K <matematica.a3k@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/klipper/kinematics/es/
Translate-URL: https://hosted.weblate.org/projects/klipper/pressure_advance/es/
Translate-URL: https://hosted.weblate.org/projects/klipper/readme/es/
Translation: Klipper/Kinematics
Translation: Klipper/Pressure_Advance
Translation: Klipper/readme
